backend/libraries/types/can.did

```diff
@@ -1756,6 +1756,12 @@ type VideoCallUpdates = variant {
     SetToSome : VideoCall;
 };
 
+type VideoCallParticipants = record {
+    participants : vec CallParticipant;
```

Comment:
Am I blind? I can't see where CallParticipant itself is defined?

Reply:
https://github.com/open-chat-labs/open-chat/blob/f6f25152e773b9a026572150eb3456122619b6b4/backend/libraries/types/can.did#L1639

Reply:
I didn't add it in this PR, it was already there